salt/states/firewalld.py

```diff
@@ -75,40 +153,27 @@ def __virtual__():
     return (False, 'firewall-cmd is not available, firewalld is probably not installed.')
 
 
-def present(name,
```

Comment:
No it isn't actually, I just did it because we now have two states that apply to firewalld `service` for service configuration and `present`/`zone` for zone configuration. I'll revert it.